app/main.py

```diff
@@ -1,6 +1,53 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
+CITY = "Paris"
+LANGUAGE = "en"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    res = requests.get(
+        URL,
+        params={
+            "key": API_KEY,
+            "q": CITY,
+            "units": "metric",
+            "lang": LANGUAGE
+        }
+    )
+    data = res.json()
+
+    if "error" in data:
+        print("Error:", data["error"]["message"])
+    else:
```

Comment:
it's better to check status_code and indeed print error

Reply:
Fixed that